src/Command/Site/UpdateCommand.php

```diff
@@ -50,15 +50,33 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 
     $commands = [];
     $commands[] = sprintf('cd %s', $this->shellPath($this->getWebRoot()));
-    $commands[] = 'drush sset system.maintenance_mode 1';
-    $commands[] = 'drush cr';
-    $commands[] = 'drush updb -y';
-    $commands[] = 'drush cim -y';
-    $commands[] = 'drush cim -y';
-    $commands[] = 'drush sset system.maintenance_mode 0';
-    $commands[] = 'drush cr';
 
-    $command = implode(' && ', $commands);
+    // Drupal 7 only;
+    if ($this->getDrupalVersion() === 7) {
+      $commands[] = 'drush -q rr';
+      $commands[] = 'drush vset maintenance_mode 1';
+      $commands[] = 'drush updb -y';
+      $this->addModuleEnableCommands($commands);
```

Comment:
we could, but that would delay the update. by running at the end, we do the important stuff first then disable modules. but yes, we could